faq/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
```

Comment:
P1:
title 태그명이 탭에 노출되니 변경해주시면 좋겠습니다.

```suggestion
  <title>판다마켓 - FAQ</title>
```